cusub_perception/localizer/include/localizer/pose_generator.h

```diff
@@ -27,12 +28,20 @@ namespace pose_generator
             virtual bool generatePose(
                 sensor_msgs::Image& image, 
                 vector<darknet_ros_msgs::BoundingBox>& bbs,
-                geometry_msgs::Pose& pose,
-                string& class_name
+                vector<localizer::Detection>& detections
```

Comment:
I like this change, super clean. Also, maybe the bbs should be a `const` vector so it's clear which is input and output?

Reply:
yea that would be nice, we don't change anything in the image or the bounding boxes

Reply:
The startgate code sorts the box list, means we have to make a copy.  Since it doesn't send the list to multiple pose generators right now doesn't really need to be const aside from clarity.